Dockerfiles/syslog-ng-dev/dev-dependencies.txt

```diff
@@ -16,6 +21,7 @@ libhiredis-dev
 libnet1-dev
 libpython-dev
 libssl-dev
+libriemann-client-dev
 libsystemd-daemon-dev
```

Comment:
I've checked this package and it's only a transitional package. It directly depends on `libsystemd-dev`. So could you change it?